pepper-apis/dsm-core/src/main/java/org/broadinstitute/dsm/db/KitRequestShipping.java

```diff
@@ -284,6 +287,8 @@ public class KitRequestShipping extends KitRequest implements HasDdpInstanceId {
     @ColumnName(DBConstants.DDP_INSTANCE_ID)
     private long ddpInstanceId;
 
+    private String collectionDate;
```

Comment:
as I got this field needs to be exported to ES, thus I believe we need to add `@ColumnName` annotation to make ES exporting work